src/box/lua/net_box.c

```diff
@@ -57,6 +57,10 @@ netbox_prepare_request(lua_State *L, struct mpstream *stream, uint32_t r_type)
 {
```

Comment:
The commit contains zero tests. Please, add them. There should be like a whole ton of them considering how much you need to test in the schema reload and on the server side with the streams management in iproto.cc.

Reply:
Add test